README.md

```diff
@@ -63,13 +63,13 @@ echo $builder->createURL("bridge.png", $params);
 // http://demos.imgix.net/bridge.png?h=100&w=100
 ```
 
-For HTTPS support, simply use the setter `setUseHttps` on the builder
+Currently, HTTPS support is available _by default_. However, if you need HTTP support, simply call `setUseHttps` on the builder:
```

Comment:
I've been removing any instances of [words to avoid](https://css-tricks.com/words-avoid-educational-writing/) in educational writing throughout our sdk documentation. It looks like I missed two instances of 'simply' in this project's README that we should get rid of.

Reply:
I like that. Pretty much all words ended in `ly` are "extra." I was just trying to match the style and voice.

Reply:
The above is outdated, see the [updated version](https://github.com/imgix/imgix-php/pull/54/commits/0717f98199556d93173778f32044d1c41a00c367).